api_tests/dry/config.ts

```diff
@@ -0,0 +1,83 @@
+import { execSync, exec } from "child_process";
+import { promisify } from "util";
+import { expect } from "chai";
+import tmp from "tmp";
+import * as fs from "fs";
+
+const execAwait = promisify(exec);
+
+const projectRoot: string = execSync("git rev-parse --show-toplevel", {
+    encoding: "utf8",
+}).trim();
+
+const defaultConfigComplete = `[network]
+listen = ["/ip4/0.0.0.0/tcp/9939"]
+[http_api.socket]
+address = "0.0.0.0"
+port = 8000
+
+[http_api.cors]
+allowed_origins = "none"
+
+[data]
+dir = "/Users/bonomat/Library/Application Support/comit"
+
+[logging]
+level = "Info"
+
+[bitcoin]
+network = "regtest"
+
+[bitcoin.connector.bitcoind]
+node_url = "http://localhost:18443/"
+
+[ethereum]
+chain_id = "0x07"
```

Comment:
Did you mean this to be "0x11" (17 in hex), like in our code or is this intentionally an uncommon chain number?